profile/README.md

```diff
@@ -0,0 +1,54 @@
+# We are NewAperio
+
+![hero images](https://user-images.githubusercontent.com/1529366/173092313-2938e23d-8e4b-4d93-87f8-c827d38c4002.png)
+
+### What we do:
```

Comment:
Maybe we can remove colons in the headings? Wondering if they're redundant since a heading implicitly introduces a topic.